### DISCUSSION TITLE
Is it possible to hide a markdown cells and code output from a Jupiter Notebook?

### DISCUSSION THREAD

**Discussion by KelSolaar** [quarto-dev/quarto-cli#3233] [Q&A]
Hello,

Title says it all! I'm wondering if is is possible to hide a markdown cell from a Jupiter Notebook?

I can hide code ones as follows:

```python
#| echo: false
```

Cheers,

Thomas

**Comment by KelSolaar** [comment]
Similarly, I would be keen to know if it is possible to also hide the output of a code cell.

**Reply by ianxek** [reply]
Yes:

1. For markdown cells, just do this:
```
::: {.content-hidden}
# This will not be displayed, completely ignored
and that also
:::
```

and for code cell:

```
#| output: false
#| echo: false
print("This code will not be displayed, nor the output")
```

**Comment by fcusimano** [comment]
You could write your markdown text in a code cell and hide its output like this:

```python
#| output: false
from IPython.display import display, Markdown
display(Markdown(f""" 
text 
"""))
```

**Comment by rahuldave** [comment]
Is there no directive we can put in the whole doc to hide markdown cells? And then selectively enable them?

**Reply by mcanouil** [reply]
Does this exist in Jupyter?
If it does exist in Jupyter, then you can use this.

Alternatively you could write a Lua filter to remove those.

**Reply by luca-heitmann** [reply]
To selectively enable content you can specifiy a when-meta parameter to ".content-hidden" like this

````
---
title: Jupyter Test
format: ipynb
hideMarkdown: true
hideCode: true
---

::: {.content-hidden when-meta="hideMarkdown"}
:::: {.cell .markdown}
### Cell 1

with markdown content
::::
:::

::: {.content-hidden when-meta="hideCode"}
:::: {.cell .code}

```
println("Cell 2 with code")
```
::::
:::
````

works also with when-profile https://quarto.org/docs/projects/profiles.html#profile-content

but of course this does not work for the whole doc

**Reply by mcanouil** [reply]
@luca-heitmann  Could you edit your code to remove the backslash?
Use more on the outer code block, i.e., four backticks.
